processing/src/main/java/org/apache/druid/segment/filter/BoundFilter.java

```diff
@@ -45,13 +46,16 @@
 import org.apache.druid.segment.column.BitmapIndex;
 import org.apache.druid.segment.vector.VectorColumnSelectorFactory;
 
+import javax.annotation.Nullable;
 import java.util.Comparator;
+import java.util.Set;
 
 public class BoundFilter implements Filter
 {
   private final BoundDimFilter boundDimFilter;
   private final Comparator<String> comparator;
   private final ExtractionFn extractionFn;
+  private final FilterTuning manualFilterTuning;
```

Comment:
Sometimes the field is called `manualFilterTuning` and sometimes it's called `filterTuning`. Could you please pick one?

Reply:
I believe it is consistently called `filterTuning` on `DimFilter` and `manualFilterTuning` on `Filter` implementations. This distinction is because everything in `DimFilter` is manual since it was provided by the json query, where as this is explictly the user set value on `Filter` implementations that was supplied by the `DimFilter`.

Reply:
renamed all to `filterTuning`